src/components/extensive/WizardForm/index.tsx

```diff
@@ -162,45 +164,64 @@ function WizardForm(props: WizardFormProps) {
   }, [selectedStepIndex]);
 
   const stepTabItems = props.steps.map((step, index) => ({
-    title: step.tabTitle || t(`Step {number}<br/> {title}`, { number: index + 1, title: step.title }),
+    title:
+      step.tabTitle ||
+      t(`Step {number}<br/> <p className="text-14-light">{title} </p>`, { number: index + 1, title: step.title }),
     done: props.tabOptions?.markDone && index < selectedStepIndex,
     disabled: props.tabOptions?.disableFutureTabs && index > selectedStepIndex,
     body: (
-      <FormStep
-        id="step"
-        formHook={formHook}
-        fields={step.fields}
-        title={step.title}
-        subtitle={step.subtitle}
-        onChange={onChange}
-        className="h-[calc(100vh-287px)] overflow-auto"
-      >
-        <FormFooter
-          className="mt-12"
-          backButtonProps={
-            !props.hideBackButton
-              ? {
-                  children: props.backButtonText || t("Back"),
-                  onClick: () => {
-                    if (selectedStepIndex > 0) {
-                      setSelectedStepIndex(n => n - 1);
-                    } else {
-                      props.onBackFirstStep();
+      <div className="h-[calc(100vh-287px)] overflow-auto">
+        {index === 0 && step.title === "Site Overview" && (
+          <div className="w-full bg-white px-16 pt-8">
+            <div className="flex gap-4 rounded-lg bg-tertiary-80 p-6">
+              <Text variant="text-16-bold" className="text-white">
+                {t(`Note: Project polygons are editable through a new geometry-focused workflow that is accessible by
+                clicking on this link.`)}
+              </Text>
+              <Button
+                variant="text"
+                className="text-14-bold nowrap whitespace-nowrap rounded-lg border-2 border-transparent bg-[#ffb88891] px-4 py-0 uppercase text-white hover:border-white"
+              >
+                edit polygon
```

Comment:
Needs a wrap in `t`

Reply:
Thanks for the feedback! I've wrapped the text in t as suggested.